app/products/calls/screens/call/call_screen.tsx

```diff
@@ -259,12 +277,16 @@ const CallScreen = (props: Props) => {
                 props.actions.muteMyself(props.call.channelId);
             }
         }
-    }, [props.call.channelId, props.currentParticipant]);
+    }, [props.call?.channelId, props.currentParticipant]);
 
     const toggleControlsInLandscape = useCallback(() => {
         setShowControlsInLandscape(!showControlsInLandscape);
     }, [showControlsInLandscape]);
 
+    if (!props.call) {
```

Comment:
is there a way to detect this early and avoid this component to be mounted if is going to render null ?

Reply:
I think this gets mounted lazily only when the user presses the button to show it. At that point it means a call is happening otherwise the UI wouldn't be there in the first place. The check is mostly needed in case the call drops or when the user leaves. Does that makes sense? Please let me know if I misunderstood your comment :p